components/apimgt/org.wso2.carbon.apimgt.impl/pom.xml

```diff
@@ -284,6 +284,12 @@
             <groupId>org.wso2.orbit.io.swagger</groupId>
             <artifactId>swagger-parser</artifactId>
         </dependency>
+        <!-- https://mvnrepository.com/artifact/io.swagger/swagger-inflector -->
+        <dependency>
+            <groupId>io.swagger</groupId>
+            <artifactId>swagger-inflector</artifactId>
+            <version>2.0.5</version>
```

Comment:
provide the version in the root pom and don't use the version here. Check how this is done for other dependencies.